src/components/Counter.js

```diff
@@ -17,13 +17,15 @@ class Counter extends Component {
         // Fill in the two button onClick methods
         // Upon clicking these buttons, the count
         // should decrement or increment accordingly
+        console.log('Counter');
+        console.log(this.props);
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment() }>
```

Comment:
Very good! We imported these methods at the top of this file from actions. Then we passed them as props to this component in our `connect` below! Great work!